include/externs.h

```diff
@@ -905,6 +905,7 @@ extern  void            query_finance           (CONTEXT);
 extern  void            query_first_name        (CONTEXT);
 extern  void            query_flags             (CONTEXT);
 extern  void            query_format            (CONTEXT);
+extern	void		      query_format_number	   (CONTEXT);
```

Comment:
Looks like tabbing issues here

Reply:
Fixed a couple of these spots. Got them all to line up properly.